src/main/java/uk/gov/dvsa/model/cvs/IVA30.java

```diff
@@ -3,17 +3,34 @@
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import uk.gov.dvsa.model.Document;
+import uk.gov.dvsa.model.cvs.certificateData.IvaFailCertificateData;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class IVA30 extends Document {
```

Comment:
The getters and setters are used by handlebars, and for the testing.

For belts and braces I think it's best to ignore anything we are not expecting, doc-gen throws an error as it will try mapping to a key/value that doesn't exist in the model.